orthoinference/src/main/java/org/reactome/orthoinference/SkipInstanceChecker.java

```diff
@@ -219,4 +327,16 @@ public static void setAdaptor(MySQLAdaptor dbAdaptor)
 	{
 		dba = dbAdaptor;
 	}
+
+	public static void setHomologueMappingFile(Map<String, String[]> homologueMappingsCopy) { homologueMappings = homologueMappingsCopy; }
+
+	public static void setEligibleFilename(String eligibleFilename)
+	{
+		eligibleFilehandle = eligibleFilename;
+	}
```

Comment:
Similar to https://github.com/reactome/data-release-pipeline/pull/104/files#r340082898